tests/virt_autotest/download_guest_assets.pm

```diff
@@ -0,0 +1,34 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2019 SUSE LLC
```

Comment:
Should be `2019` :)

Reply:
uh, correct it. did not know what it means :)